### ISSUE TITLE
can you split the image files into smaller files

### ISSUE THREAD

**Issue by AiliAili** [entitize/Fakeddit#9]
Hi entitize,

Thanks for releasing the dataset.

I've tried to download the image files from "https://drive.google.com/file/d/1cjY6HsHaSZuLVHywIxD5xQqng33J5S2b/view"
But the file is so big, I guess it takes a long time to download it. (I've encountered multiple times of failure in downloading this file.)
Then I tried to download it via python, but encountered the following:

"Access denied with the following error:

        Too many users have viewed or downloaded this file recently. Please
        try accessing the file again later. If the file you are trying to
        access is particularly large or is shared with many people, it may
        take up to 24 hours to be able to view or download the file. If you
        still can't access a file after 24 hours, contact your domain
        administrator. 

       You may still be able to access the file from the browser:

         https://drive.google.com/uc?id=1cjY6HsHaSZuLVHywIxD5xQqng33J5S2b "
, even after I changed my IPs.

The downloading process can be interrupted by the network connection at any time, leading to a failure downloading process.
If possible, could you please provide multiple image files in a smaller size (such as 10GB)? In this way, we just need to download the failed files, if the internet is disconnected at some time.

**Comment by sharonlevy** [comment]
Hi AiliAili,

We have split the original data into smaller files and re-uploaded them to the google drive folder. They are named public_image_set_* and numbered from 1-19. You should be able to access it here: https://drive.google.com/drive/folders/1vNFLeuf8__lJLSGdpq1TyMeOLSUWv_BD?usp=sharing

**Comment by AiliAili** [comment]
Hi sharonlevy,

Thank you so much for offering the splits of the image file.
Finally, I downloaded it successfully. 
However, I find some image files is not a valid file. There are roughly 500 such files in the provided dataset. I tried to re-download these files from the urllinks in the tsv file. But it turned out that these URLs are not accessible.

Should I just remove these files from the dataset? I've provided a list of ids with corrupted image files so you verify on your end.

Another question, can we see other people's submission rank in the competition page "https://competitions.codalab.org/competitions/25337"? I didn't see any reported numbers on this website.

Thank you very much!
